### PR TITLE
First PR with Models Project

### PR DESCRIPTION
**Pull request by dadheech115**
This PR contains work for :
#1 Models project file
#2 Base models
#3 Generated Models
#4 Podspec file
#5 Ruby script to add file

### CODE REVIEW

File: MSGraphModelsTests/MSGraphModelsTests.m [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+@end
+
+@implementation MSGraphModelsTests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown {
```
Comment (deepak2016) [inline MSGraphModelsTests/MSGraphModelsTests.m]: remove unused methods

File: MSGraphModelsTests/Base Models/NSDate+MSSerializationTests.m [hunk 2]
```diff
@@ -0,0 +1,58 @@
+//
+// Copyright (c) Microsoft Corporation. All Rights Reserved. Licensed under the MIT License. See License in the project root for license information.
```
Comment (deepak2016) [inline MSGraphModelsTests/Base Models/NSDate+MSSerializationTests.m]: remove space from test folder's name

File: BaseModels/MSDuration.m [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    minutes = fmod(minutes, 60);
+    interval = interval - minutes*60;
+
+    hours = interval/ (60*60);
+    hours = fmod(hours , 24);
+    interval = interval - hours*60*60;
+
+    days = interval/ (24*60*60);
+
+    NSLog(@"");
```
Comment (deepak2016) [inline BaseModels/MSDuration.m]: what is this log statement?

Comment (deepak2016) [reply BaseModels/MSDuration.m]: ?

Comment (dadheech115) [reply BaseModels/MSDuration.m]: Removed it.

File: MSGraphModelsTests/MSGraphModelsTests.m (hunk 1 above)
Comment (dadheech115) [reply MSGraphModelsTests/MSGraphModelsTests.m]: Done.

**Pull request merged**